rosidl_generator_rs/cmake/rosidl_generator_rs_generate_interfaces.cmake

```diff
@@ -242,25 +231,21 @@ foreach(_typesupport_impl ${_typesupport_impls})
 
   if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
     install(
-      DIRECTORY "${_output_path}/rust"
+      DIRECTORY "${_output_path}/Rust"
```

Comment:
Why the rename?

Reply:
Well, I'd always capitalize Rust, so it just seem tidier to me. But we could also use "rs" as an abbreviation, or revert that.

Reply:
Rust in documents and comments I'd say yes always, but for paths I don't see why. For example, Cargo uses `.cargo` in lower case for its path. Having paths in lowercase seems more convenient to me.

Reply:
I guess when it's lowercase I read it as a verb, "to rust". But I'll revert it.

Reply:
Done.